File: javascript/tests/roman_to_integer_test.js
import {assertEquals} from "@std/assert";
import {romanToInt} from "@/roman_to_integer.js";

Deno.test(function exampleCases() {
    assertEquals(romanToInt("III"), 3);
    assertEquals(romanToInt("LVIII"), 58);
    assertEquals(romanToInt("MCMXCIV"), 1994);
});

File: java/src/test/java/com/jamesansley/RomanToIntegerTest.java
package com.jamesansley;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class RomanToIntegerTest {
    @Test
    public void exampleCases() {
        RomanToInteger solution = new RomanToInteger();

        assertEquals(3, solution.romanToInt("III"));
        assertEquals(58, solution.romanToInt("LVIII"));
        assertEquals(1994, solution.romanToInt("MCMXCIV"));
    }
}
